doc/topics/packaging/index.rst

```diff
@@ -10,10 +10,78 @@ Relenv onedir packaging
 Starting in 3006, only onedir packaging will be available. The 3006 onedir packages
 are built with the `relenv <https://github.com/saltstack/relative-environment-for-python>`_ tool.
 
+
+Docker Containers
+=================
+We use docker containers to build our packages. If you are building your own packages you can use
```

Comment:
**Suggestion (style, non-blocking):** I'd recommend saying "The Salt Project uses..." rather than "We use." Our project's parent style guide (Google Developer's) recommends avoiding use of the first person in documentation. https://developers.google.com/style/person